README.md

```diff
@@ -1,6 +1,6 @@
 # IMAPClient [![Coverage Status](https://coveralls.io/repos/github/hpi-swa-teaching/IMAPClient/badge.svg?branch=develop)](https://coveralls.io/github/hpi-swa-teaching/IMAPClient?branch=develop)
  
-Welcome to our IMAP Client project. We are a team of 4th semester IT-Systems Engineering students and are happy to take over this project for the sixth iteration.
+Welcome to our IMAP Client project. We are a team of 4th semester IT-Systems Engineering students and are happy to take over this project for the seventh iteration.
```

Comment:
are we already seventh?

Reply:
This refers to the next team 😅 The team before us also updated it that way, so I did too.

Reply:
I'm pretty sure, someone from our team updated the Readme this year (our names were added), but ok, it can stay that way